service/service.go

```diff
@@ -363,3 +385,43 @@ func pdataFromSdk(res *sdkresource.Resource) pcommon.Resource {
 	}
 	return pcommonRes
 }
+
+func disableHighCardinalityMetrics() []config.View {
```

Comment:
looks OK for now, but if we expect this list to grow before we remove the gate, it might make sense to externalize this somehow?

Reply:
This is the purpose of #11754